### PR TITLE
refactor: move default actions to TILE_DEFAULTS

### PR DESCRIPTION
**Pull request by akloeckner**
also introduce the option to disable actions by setting `action: false` or `secondaryAction: false`
also show "clickable" tick not for fixed tile types, but for tiles with `action` defined

### CODE REVIEW

File: scripts/controllers/main.js [hunk 1]
```diff
@@ -80,66 +80,29 @@ App.controller('Main', function ($scope, $timeout, $location, Api) {
    window.addEventListener('resize', debounce(innerHeightToCSS, 250));
 
    $scope.entityClick = function (page, item, entity) {
-      if (typeof item.action === 'function') {
-         return callFunction(item.action, [item, entity]);
+      if (item.action === false) {
+         return null;
```
Comment (rchl) [inline scripts/controllers/main.js:84]: I think just `return` would be more correct. Returning `null` for `ng-click` seems like has no special meaning so `undefined` would work just fine also.

File: scripts/globals/constants.js [hunk 2]
```diff
@@ -294,7 +339,63 @@ export const TILE_DEFAULTS = {
...
+      },
+   },
+   [TYPES.LOCK]: {
+      action (item, entity) {
+         return this.$scope.toggleLock(item, entity);
+      },
+   },
+   [TYPES.POPUP]: {
+      action (item, entity) {
+         return this.$scope.openPopup(item, entity);
```
Comment (rchl) [inline scripts/globals/constants.js:373]: This one also took the third argument `item.popup` before.

**Review by rchl — CHANGES_REQUESTED** [review]

File: scripts/globals/constants.js (hunk 2 above)
Comment (akloeckner) [reply scripts/globals/constants.js:373]: Yes. In fact, that was redundant, as the `openPopup` defaults to `item.popup` itself anyway. I dropped when I was trying to find a simpler syntax like
```
action: $scope.openPopup,
```
But I failed to find one. If you can think of one, let me know, please.

Until then, I'll re-introduce the third argument for clarity. Despite its redundancy.

**Comment by rchl** [comment]
Thanks.

**Pull request merged**